keps/sig-storage/1855-csi-driver-service-account-token/README.md

```diff
@@ -245,6 +295,25 @@ Option 1 is adopted. See discussion
 - **Will enabling / using this feature result in non-negligible increase of
   resource usage (CPU, RAM, disk, IO, ...) in any components?** no.
 
+### Troubleshooting
+
+- **How does this feature react if the API server and/or etcd is unavailable?**
+  `RequiresRepublish` will continue to function but `TokenRequests` will fail.
+
+- **What are other known failure modes?**
+
+  - Failed to fetch token
+
+    - Detection: Chec mount failure events or kubelet log.
```

Comment:
how do I know to check them?  The only listed metric is `storage_operation_duration_seconds`.

Reply:
> how do I know to check them? The only listed metric is `storage_operation_duration_seconds`.

I see, status is one of the labels.

Reply:
mentioned `status` under the question `What specific metrics should inform a rollback?`